tests/test_integrations/test__init__.py

```diff
@@ -54,13 +54,13 @@ def test_create_dataprovider(transfer):
                 conn_id="fivetran_default",
                 connector_id="filing_muppet",
                 group=Group(name="test_group"),
-                connector=Connector(
+                connector=FivetranConnector(
                     service="s3",
                     config={},
                     connector_id=None,
                     connect_card_config={"connector_val": "test_connector"},
                 ),
-                destination=Destination(
+                destination=FivetranDestination(
```

Comment:
Same as - https://github.com/astronomer/apache-airflow-provider-transfers/pull/62/files#r1175090418

Reply:
This is taken care of in _init__ of the `FivetranIntegrations` class. This test here is used to fetch the ingestor_option_class class. In this scenario, the dict is already resolved to nested `FiveTranOptions` class so it should be as it is.

Reply:
Nice!